aspnetcore/security/enforcing-ssl.md

```diff
@@ -356,12 +356,21 @@ The Windows Subsystem for Linux (WSL) generates an HTTPS self-signed cert. To co
 * Run the following command to export the WSL-generated certificate:
 
   ```
-  dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <cryptic-password>
+  dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <password>
```

Comment:
We need to modify this entire section, we don't follow the same approach for WSL than for docker. One is persistent, the other is not.

The text here is wrong in a couple of places. The certificate doesn't come from WSL it comes from the windows host. The command makes it look like you export it from within WSL and that's not the case. Also, the certificate doesn't need to be exported to `%USERPROFILE%\.aspnet\https\aspnetapp.pfx` just the current folder works fine.

`%USERPROFILE%\.aspnet\https\<<app-name>>` is a special location used by docker tooling, we should leave that alone.
```
* Run the following command to export the WSL-generated certificate:
 ```

Reply:
@javiercn @Rick-Anderson 
Hi guys, I'm very glad to see this is being addressed and look forward to the `--import` feature in preview 8!

I just want to point out that the text is still wrong, it still says "WSL-generated" instead of "Windows-generated." This was very confusing for me when I was reviewing the docs a few days ago.

Thanks guys!

Reply:
Oh, also it needs to be `export` for the environment variables.